src/current/v24.1/read-committed.md

```diff
@@ -25,18 +25,7 @@ If your workload is already running well under `SERIALIZABLE` isolation, Cockroa
 
 ## Enable `READ COMMITTED` isolation
 
-To make `READ COMMITTED` isolation available to use on a cluster, enable the following [cluster setting]({% link {{ page.version.version }}/cluster-settings.md %}) in the SQL shell:
-
-{% include_cached copy-clipboard.html %}
-~~~ sql
-SET CLUSTER SETTING sql.txn.read_committed_isolation.enabled = 'true';
-~~~
-
-After you enable the cluster setting, you can set `READ COMMITTED` as the [default isolation level](#set-the-default-isolation-level-to-read-committed) or [begin a transaction](#set-the-current-transaction-to-read-committed) as `READ COMMITTED`.
-
-{{site.data.alerts.callout_info}}
-If the cluster setting is not enabled, `READ COMMITTED` transactions will run as `SERIALIZABLE`.
-{{site.data.alerts.end}}
+By default, the `sql.txn.read_committed_isolation.enabled` [cluster setting]({% link {{ page.version.version }}/cluster-settings.md %}) is enabled. If the cluster setting is `false`, `READ COMMITTED` transactions will run as `SERIALIZABLE`.
```

Comment:
Do we want to talk somewhere about how a user can determine whether their application was previously asking for READ COMMITTED and getting SERIALIZABLE? This is now possible with the observability that Rafi added in https://github.com/cockroachdb/cockroach/pull/118497.

The thinking is that users will want to check this metric before performing the v24.1 upgrade to determine whether the upgrade will change the behavior of their application. For some users, that may be a welcomed change. For others, it could cause issues.

Reply:
I'm going to add this in a separate PR as I need Rafi to review the docs for those metrics. Thanks for calling this out.

Reply:
Actually just made another commit here. @rafiss (and anyone else) would you mind reviewing? https://github.com/cockroachdb/docs/pull/18522/commits/ab334f79c2560323d64f04a4a98db5691bbaa8ac